.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+cache: npm
+
+node_js:
+  - "6"
+  - "8"
+  - "10"
+
+branches:
+  only:
+    - master
```

Comment:
I can never remember, does this block pull requests builds?

Reply:
There are two types of builds we can do: branch builds, or PR builds. If we don't set this, and we allow both in the UI, then our PRs will trigger both builds at once (duplicate). By setting this, we only run builds on pushes to master or on PRs that are opened. We don't trigger builds on pushed branches that are not used to open a PR.

Reply:
That's not confusing at all